cll1-v3.txt

```diff
@@ -115,8 +115,8 @@ ID|PSET|ALT_PRONOUNCE|PRONOUN|VERB|GENDER|SYLLABARY|PRONOUNCE|ENGLISH|INTRO NOTE
 
 ||||||ᎭᏢ ᎥᏍᎩᎾ; ᎭᏢ ᎥᏍᎩ; ᎭᏢ ᏍᎩᎾ; ᎭᏢ ᎾᏍᎩ|ha:dlv v̀:sgina; ha:dlv v̀:sgi; ha:dlv sgina; ha:dlv nà:sgi|Where is he? Where are they?|||
 
-||||||ᎭᏢ Ꮎ ꮅᎬᎢ|Ha:dlv na hlʔgv̋:ʔi?|Where is the tree?|||
-||||||ᎭᏢ Ꮎ ᏕꮅᎬᎢ|Ha:dlv na dě:hlʔgv̋:ʔi?|Where are the trees?|||
+||||||ᎭᏢ Ꮎ ꮅᎬᎢ|Ha:dlv na hlʔgv̋:ʔi?; Ha:dlv na hluʔgv̋:ʔi?; Ha:dlv na dluhgv̋:ʔi?; Ha:dlv na hlʔgv̋?; Ha:dlv na hluʔgv̋?;Ha:dlv na dluhgv̋?|Where is the tree?|||
```

Comment:
What's up with the lowercase Ꮅ? I have seen it a few times. Also, I believe it should be a Ꮱ. I will try to get this in my syllabary fixups